scripts/bdc/get_bdc_studies_md_from_gen3.py

```diff
@@ -0,0 +1,269 @@
+                
+                partial_list = partial_list_response.json()
+                complete_list.extend(partial_list)
+                
+                if len(partial_list) < download_limit:
+                    break
+                    
+                offset += download_limit
+                
+            except requests.exceptions.RequestException as e:
+                self.logger.error(f"Error downloading from Gen3: {e}")
+                raise
+        
+        if len(complete_list) != len(set(complete_list)):
+            duplicate_ids = sorted([ident for ident, count in Counter(complete_list).items() if count > 1])
+            self.logger.warning(f"Found duplicate discovery_metadata: {duplicate_ids}")
+        
+        return complete_list
+
+    def download_raw_data(self):
+        """Download and save raw data from Gen3"""
+        self.raw_file = os.path.join(self.output_dir, "gen3_studies_raw.csv")
+        self.logger.info(f"Starting raw data download. Output file: {self.raw_file}")
+        
+        # Download metadata
+        base_url = 'https://gen3.biodatacatalyst.nhlbi.nih.gov/'
+        mds_discovery_metadata_url = urllib.parse.urljoin(
+            base_url,
+            f'/mds/metadata?_guid_type=discovery_metadata'
+        )
+        
+        discovery_list = self.download_gen3_list(mds_discovery_metadata_url)
+        self.logger.info(f"Downloaded {len(discovery_list)} discovery_metadata entries")
+        
+        sorted_study_ids = sorted(discovery_list)
+        
+        # Write to CSV
+        with open(self.raw_file, 'w', newline='') as csvfile:
+            csv_writer = csv.DictWriter(csvfile, fieldnames=[
+                'Accession', 'Consent', 'Study Name', 'Program', 
+                'Last modified', 'Notes', 'Description'
+            ])
+            csv_writer.writeheader()
+            
+            for study_id in sorted_study_ids:
+                # Reset variables
+                study_name = ''
+                program_names = []
+                description = ''
+                notes = ''
+                last_modified = str(datetime.now().date())
+                
+                # Get study info
+                url = urllib.parse.urljoin(base_url, f'/mds/metadata/{study_id}')
+                try:
+                    study_info_response = requests.get(url)
+                    study_info_response.raise_for_status()
+                    study_info = study_info_response.json()
+                    
+                    if 'gen3_discovery' in study_info:
+                        gen3_discovery = study_info['gen3_discovery']
+                        
+                        # Handle study name
+                        if 'full_name' in gen3_discovery:
+                            study_name = gen3_discovery['full_name']
+                            notes += f"Name: {gen3_discovery.get('name', '')}, short name: {gen3_discovery.get('short_name', '')}.\n"
```

Comment:
I'm not sure why you're writing to `notes` here. I would suggest use the if-elif code to figure out a `study_name` (and optionally a `short_name` if that's useful to you), and then writing both fields to `notes` at the end.